scripts/infra-deploy.sh

```diff
@@ -54,6 +54,8 @@ if [ $EXPORTS_SET = 1 ] ; then
   exit 1
 fi
 
+echo "Branch name: $BRANCH_NAME"
```

Comment:
This may be a temp echo if not how about we incorporate it into line 63... in my follow on branch I am adding to line 63 the derived name of the terraform workspace which maybe enough